integration-tests/kamelet/src/main/java/org/apache/camel/quarkus/component/kamelet/it/KameletRoutes.java

```diff
@@ -62,6 +62,9 @@ public void configure() throws Exception {
         from("direct:chain")
                 .to("kamelet:echo/1?prefix=Camel Quarkus&suffix=Chained")
                 .to("kamelet:echo/2?prefix=Hello&suffix=Route");
+
+        from("direct:kamelet-location-at-runtime")
+                .kamelet("upper?location=file:src/test/resources/upper-kamelet.xml");
```

Comment:
Is that location going to work on `quarkus-platform`?

Reply:
Yeah good catch, it should not work on platform. Maybe it is feasible to find another way, worst case the test may be removed.

Reply:
The test is now using classpath, should be ok on quarkus-platform.